circle.yml

```diff
@@ -1791,14 +1791,6 @@ jobs:
       - run:
           name: Build
           command: yarn workspace @cypress/angular build
-      - run:
```

Comment:
No tests for `npm/angular`. To test this, we would need to install all of the webpack deps for Angular so all of the mounting tests are found in the `system-tests/project-fixtures/angular/src/app/mount.cy.ts` which are run in `npm-webpack-dev-server` e2e tests.